src/layers/sds/cpm_translation/__init__.py

```diff
@@ -71,24 +56,34 @@ def translate(
 
     if object_class == NhsAccreditedSystem.OBJECT_CLASS:
         instance = NhsAccreditedSystem.construct(**obj)
-        translate_kwargs = dict(
+        devices = nhs_accredited_system_to_cpm_devices(
             nhs_accredited_system=instance,
             questionnaire=spine_device_questionnaire,
             _questionnaire=_spine_device_questionnaire,
             _trust=_trust,
-            repository=repository,
         )
-        translations = ACCREDITED_SYSTEM_TRANSLATIONS
-    else:
+    elif object_class == NhsMhs.OBJECT_CLASS:
         instance = NhsMhs.construct(**obj)
-        translate_kwargs = dict(
+        nhs_mhs_to_cpm_device(
             nhs_mhs=instance,
             questionnaire=spine_endpoint_questionnaire,
             _questionnaire=_spine_endpoint_questionnaire,
             _trust=_trust,
             repository=repository,
         )
-        translations = MESSAGE_HANDLING_SYSTEM_TRANSLATIONS
+    elif object_class == SdsDeletionRequest.OBJECT_CLASS:
+        instance = SdsDeletionRequest.construct(**obj)
+        devices = delete_devices(
+            deletion_request=instance,
+            questionnaire_ids=[
+                spine_endpoint_questionnaire.id,
+                spine_device_questionnaire,
```

Comment:
think this is missing the id?

Reply:
thanks! not actually run/written any tests or anything yet...

Reply:
<img src="https://media0.giphy.com/media/q5j5JpviCUDEk/giphy.gif"/>